README.md

```diff
@@ -93,6 +95,16 @@ Use the [App Manifest file](https://github.com/seratch/deepl-for-slack/blob/mast
 * Click **Open DeepL Tool**
 * Enter a text and the language on the modal
 
+### Note on auto translate feature
+* If activated this slack app (when added to a channel) will automatically translate any incoming message.
+* It will not automatically translate messages from other apps.
+* Due to limitations of the DeepL API the slack app will use the first language as default source language.
+  This will result in messages written in the other language to be translated twice (using up more cloud credits).  
```

Comment:
This is because you translate this app's messages as well. If you improve the code, this won't happen for sure.

Reply:
bot_message subtype is exlcuded from any translation already.
this could cause problems especially if two translation bots are added to the channel.
the double translation is a different topic (see first comment on this pull request, it's inherent to DeepL API)